tests/menus/context_menu_comonent.test.ts

```diff
@@ -412,9 +422,11 @@ describe("Context Menu internal tests", () => {
     const menuItem = fixture.querySelector(".o-menu div[data-name='paste_special']");
     expect(menuItem?.classList).not.toContain("o-menu-item-active");
     triggerMouseEvent(menuItem, "mouseover");
+    jest.runAllTimers();
```

Comment:
use your helper `mouseOverMenuElement` ? You can use the `getTarget()` helpers from `dom_helpers.ts` to make it work with eiter an Element or a slector string